tests/stdlib/tsetutils.nim

```diff
@@ -2,13 +2,39 @@ discard """
   targets: "c js"
 """
 import std/setutils
-
+type 
+  Colors = enum
+    red, green = 5, blue = 10
+  Bar = enum
+    bar0 = -1, bar1, bar2
 template main =
-  doAssert "abcbb".toSet == {'a', 'b', 'c'}
-  doAssert toSet([10u8, 12, 13]) == {10u8, 12, 13}
-  doAssert toSet(0u16..30) == {0u16..30}
-  type A = distinct char
-  doAssert [A('x')].toSet == {A('x')}
-  
+  block: # toSet
+    doAssert "abcbb".toSet == {'a', 'b', 'c'}
+    doAssert toSet([10u8, 12, 13]) == {10u8, 12, 13}
+    doAssert toSet(0u16..30) == {0u16..30}
+    type A = distinct char
+    doAssert [A('x')].toSet == {A('x')}
+
+  block: # fullSet
+    doAssert fullSet(Colors) == {red, green, blue}
+    doAssert fullSet(char) == {0.chr..255.chr}
+    doAssert fullSet(Bar) == {bar0, bar1, bar2}
+
+  block: # not
+    doAssert {red, blue}.not == {green}
+    doAssert (not {red, green, blue}).card == 0
+    doAssert {bar0}.not == {bar1, bar2}
+    doAssert {range[0..10](0), 1, 2, 3}.not == {range[0..10](4), 5, 6, 7, 8, 9, 10}
+    doAssert {'0'..'9'}.not == {0.char..255.char} - {'0'..'9'}
+
+  block: # bugs
```

Comment:
https://github.com/nim-lang/Nim/pull/17081 fixed this bug, so these tests can be re-integrated in their respective blocks